pyhmy/rpc/account.py

```diff
@@ -0,0 +1,204 @@
+from .request import (
+    _default_endpoint,
+    _default_timeout,
+    rpc_request
+)
+
+from .blockchain import (
+    get_sharding_structure
+)
+
+
+def get_balance(address, endpoint=_default_endpoint, timeout=_default_timeout) -> int:
+    """
+    Get current account balance
+
+    Parameters
+    ----------
+    address: str
+        Address to get balance for
+    endpoint: :obj:`str`, optional
+        Endpoint to send request to
+    timeout: :obj:`int`, optional
+        Timeout in seconds
+
+    Returns
+    -------
+    int
+        Account balance in ATTO
+    """
+    params = [
+        address,
+        'latest'
+    ]
+    return int(rpc_request('hmy_getBalance', params=params, endpoint=endpoint, timeout=timeout)['result'], 0)
```

Comment:
prefer explicitly stating that it is base 10: `int(...,base=10)` or drop the base arg. 

Ditto for further usages of int in the PR